sdk/cosmos/azure-spring-data-cosmos/src/main/java/com/azure/spring/data/cosmos/core/query/CriteriaType.java

```diff
@@ -123,7 +123,12 @@ public enum CriteriaType {
     /**
      * Array contains
      */
-    ARRAY_CONTAINS("ARRAY_CONTAINS");
+    ARRAY_CONTAINS("ARRAY_CONTAINS"),
+    
+    /**
+     * String equals
+     */
+    STRING_EQUAL("STRINGEQUALS");
```

Comment:
should it be `STRING_EQUALS `

Reply:
NO - this one is correct - it needs to be the SQL function name - https://docs.microsoft.com/en-us/azure/cosmos-db/sql/sql-query-stringequals

Reply:
oh I mean the enum name: `STRING_EQUAL` -> `STRING_EQUALS`